script/release/release.js

```diff
@@ -354,13 +354,18 @@ async function makeRelease (releaseToValidate) {
   } else {
     let draftRelease = await getDraftRelease();
     uploadNodeShasums();
```

Comment:
Should we move this too? I think this change makes sense on paper and as such we probably shouldn't upload _anything_ until the release is validated?

Reply:
Makes sense.